enterprise_access/apps/api_client/lms_client.py

```diff
@@ -372,3 +373,26 @@ def get_pending_enterprise_group_memberships(self, enterprise_group_uuid):
             logger.exception('Incorrect data received from LMS. [%s]', url)
 
         return None
+
+    def update_learner_status(self, enterprise_group_uuid, learner_email):
+        """
+        Partially updates learners
+
+        Arguments:
+            enterprise_group_uuid (str): uuid of the enterprise group uuid
+            learner_email (str): email for learner
+
+        """
+        try:
+            url = f'{self.enterprise_group_membership_endpoint}' + (
+                f'{enterprise_group_uuid}/learners/')
+            payload = {'learner': learner_email,
+                       'status': GROUP_MEMBERSHIP_EMAIL_ERROR_STATUS,
+                       'errored_at': localized_utcnow()}
+            response = self.client.patch(url, data=payload)
+            return response.json()
+        except requests.exceptions.HTTPError:
+            logger.exception('Failed to update data to LMS. URL: [%s].', url)
```

Comment:
I feel like we could make this error message better, like "failed to update group membership status"